### DISCUSSION TITLE
【不具合】 編集リクエストの差分表示で\nがそのまま表示されずに改行処理されてしまう

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#909] [要望・アイデア・不具合]
編集リクエストでの差分表示で、

```diff
- \nが改行記号です。
+ `\n`が改行記号です。
```

と表示されて欲しいのに、以下のように改行処理された表示になります。

![image](https://github.com/user-attachments/assets/4766d133-d011-453d-88ff-3e2d92130939)

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご報告ありがとうございます。

確認したところ、編集リクエスト作成時の差分表示に限らず、編集履歴ページなどの差分表示でも同様の問題が起きているようです。
詳しく調査して、修正が可能であれば対応します！

**Reply by shiracamus** [reply]
これはまだ修正されていないんでしたっけ？

**Reply by P-SiZK** [reply]
@shiracamus 
ご報告お待たせしてしまい申し訳ございません :bow:
調査を進めた結果、現在使用しているライブラリ側の問題であることが分かりました。

そのため、直近の開発優先度などから修正対応については改めて検討予定です。
ご不便をおかけし恐縮ですが、何卒ご理解のほどよろしくお願いいたします。

**Reply by P-SiZK** [reply]
@shiracamus 
お待たせしました、その後の対応についてご連絡いたします。

社内で改めて検討した結果、今回はライブラリ側での修正を待つことにいたしました。
該当の問題については、ライブラリ側に[修正の Pull Request](https://github.com/samg/diffy/pull/126) を送信しており、既に merge されております。
今後、ライブラリの新しいバージョンがリリースされ次第、アップデート作業を行う予定です。

何卒ご理解いただけますと幸いです :bow:

**Reply by yuta0709** [reply]
@shiracamus 
お待たせしました🙇‍♂️

先日お伝えしていたライブラリの新バージョンがリリースされたため、ライブラリをアップデートし、ご報告いただいた差分表示の問題を修正いたしました。
編集リクエストでの差分表示はもちろん、編集履歴ページやその他の差分表示箇所においても、正しく表示されるようになっております。

この度はご報告をいただきありがとうございました！
もし他にもお気づきの点がございましたら、お気軽にお知らせください。
